<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scanner</class>
 <widget class="QWidget" name="Scanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="4" column="0">
    <widget class="QScrollArea" name="scrollCameraArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>423</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="spinLaserStep">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Stopień obrotu lasera</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.187500000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="buttonLaserRight">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Obróć laser w prawo</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Zasoby.qrc">
          <normaloff>:/Icons/right.png</normaloff>:/Icons/right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="buttonLaserZeroAngle">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Zeruj aktualny kąt</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Zasoby.qrc">
          <normaloff>:/Icons/zeruj.png</normaloff>:/Icons/zeruj.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="labelLaserAngle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0°</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="buttonLaser">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Zasoby.qrc">
          <normaloff>:/Icons/laser.png</normaloff>:/Icons/laser.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="labelSliderPos">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QSlider" name="sliderScanAccurancy">
           <property name="toolTip">
            <string>Dokładność obrotu</string>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>13</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="13">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonConnected">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Status połączenia</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/Icons/notconnected.png</disabledoff>
          <disabledon>:/Icons/connected.png</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="buttonLaserLeft">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Obróć laser w lewo</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Zasoby.qrc">
          <normaloff>:/Icons/left.png</normaloff>:/Icons/left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QCheckBox" name="checkHandRotate">
        <property name="text">
         <string>Ręczny obrót przedmiotu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="buttonScan">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>SKANUJ</string>
     </property>
     <property name="icon">
      <iconset resource="Zasoby.qrc">
       <normaloff>:/Icons/scan.png</normaloff>:/Icons/scan.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QToolButton" name="buttonCameraStop">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Wyłącz kamerę</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Zasoby.qrc">
          <normaloff>:/Icons/stop.png</normaloff>:/Icons/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="inProgress">
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Kamera jest uruchomiona</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QComboBox" name="comboCamera">
        <property name="toolTip">
         <string>Lista kamer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QSpinBox" name="spinCameraFps">
           <property name="toolTip">
            <string>FPS</string>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>FPS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QToolButton" name="buttonCameraRefresh">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Odśwież listę kamer</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Zasoby.qrc">
          <normaloff>:/Icons/refresh.png</normaloff>:/Icons/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="buttonCameraStart">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Uruchom kamerę</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Zasoby.qrc">
          <normaloff>:/Icons/start.png</normaloff>:/Icons/start.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboCameraDefinition">
        <property name="toolTip">
         <string>Rozdzielczość kamery</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>640x480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>854x480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>960x720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280x720</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spinCameraFov">
        <property name="toolTip">
         <string>FOV</string>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QComboBox" name="comboCameraSize">
        <item>
         <property name="text">
          <string>100%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10%</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="19">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="spinCameraDistance">
        <property name="toolTip">
         <string>Ogległość lasera od kamery</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="value">
         <double>28.300000000000001</double>
        </property>
       </widget>
      </item>
      <item row="0" column="20">
       <widget class="QCheckBox" name="checkShowCentre">
        <property name="text">
         <string>Pokaż środek</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QListWidget" name="listLog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
